hortonworks/cloudbreak
# CB-13330: Allow Datahub upgrade when ephemeral storage based instances are used

Testing steps
- Create an environment with DataLake 7.2.9
- Create a DataHub with `7.2.9 - Data Mart with Apache Impala and Hue` template.
- In the DataHub's `upgrade` tab the available upgrades should be listed.
- Runtime upgrade to 7.2.10 should be successful
